ckanext/pages/i18n/es/LC_MESSAGES/ckanext-pages.po

```diff
@@ -0,0 +1,277 @@
+#, python-format
+msgid "Unauthorized to read organization %s"
+msgstr "Organización no autorizada para leer %s"
+
+#: ckanext/pages/controller.py:64 ckanext/pages/controller.py:309
+msgid "Page Not Found"
+msgstr "Página no encontrada"
+
+#: ckanext/pages/controller.py:66 ckanext/pages/controller.py:311
+msgid "Unauthorized to delete page"
+msgstr "No autorizado para eliminar la página"
+
+#: ckanext/pages/controller.py:68 ckanext/pages/controller.py:123
+#: ckanext/pages/controller.py:313
+msgid "Group not found"
+msgstr "Grupo no encontrado"
+
+#: ckanext/pages/controller.py:125
+#, python-format
+msgid "Unauthorized to read group %s"
+msgstr "Grupo no autorizado para leer %s"
+
+#: ckanext/pages/controller.py:240
+msgid "ERROR: View not found {view_id}"
+msgstr "ERROR: vista no encontrada {view_id}"
+
+#: ckanext/pages/controller.py:248
+msgid "Your browser does not support iframes."
+msgstr "Su navegador no soporta iframes."
+
+#: ckanext/pages/controller.py:354
+msgid "Unauthorized to create or edit a page"
+msgstr "No autorizado para crear o editar una página"
+
+#: ckanext/pages/controller.py:373
+msgid "Only Posting is availiable"
+msgstr "Solo publicaciones está disponible"
+
+#: ckanext/pages/controller.py:378
+#, python-format
+msgid "Unauthorized to upload file %s"
+msgstr "No autorizado para subir archivo %s"
+
+#: ckanext/pages/theme/templates_group/group/read_base.html:5
+#: ckanext/pages/theme/templates_main/ckanext_pages/edit_base.html:3
+#: ckanext/pages/theme/templates_main/ckanext_pages/group_page.html:3
+#: ckanext/pages/theme/templates_main/ckanext_pages/group_page_edit.html:3
+#: ckanext/pages/theme/templates_main/ckanext_pages/group_page_list.html:9
+#: ckanext/pages/theme/templates_main/ckanext_pages/group_page_list.html:12
+#: ckanext/pages/theme/templates_main/ckanext_pages/organization_page.html:3
+#: ckanext/pages/theme/templates_main/ckanext_pages/organization_page_edit.html:3
+#: ckanext/pages/theme/templates_main/ckanext_pages/organization_page_list.html:9
+#: ckanext/pages/theme/templates_main/ckanext_pages/organization_page_list.html:12
+#: ckanext/pages/theme/templates_main/ckanext_pages/pages_list.html:10
+#: ckanext/pages/theme/templates_main/header.html:5
+#: ckanext/pages/theme/templates_organization/organization/read_base.html:5
+msgid "Pages"
+msgstr "Páginas"
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog.html:34
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog_edit_base.html:9
+#: ckanext/pages/theme/templates_main/ckanext_pages/blog_list.html:10
+#: ckanext/pages/theme/templates_main/header.html:10
+msgid "Blog"
+msgstr "Blog"
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base.html:15
+msgid "What are Pages?"
+msgstr "¿Qué son páginas?"
+
+#: ckanext/pages/theme/templates_main/ckanext_pages/base.html:18
+msgid ""
+" <p>Pages are are simple way to add extra pages to your ckan instance. "
+"You can assign an order to them.</p> "
+msgstr ""
+
```

Comment:
Should be: Páginas es una forma de añadir adicional páginas a tu instancia de CKAN.